R/pkg/R/mllib.R

```diff
@@ -377,32 +476,9 @@ setMethod("predict", signature(object = "KMeansModel"),
             return(dataFrame(callJMethod(object@jobj, "transform", newData@sdf)))
           })
 
-#' Fit a Bernoulli naive Bayes model
+#' Saves a fitted MLlib model to the input path
```

Comment:
We need have an agreement on the style of the title for a method. That is, whether or not to add "s" suffix to a verb. Seems R's tradition is not. for example, the title for "lapply":
    Apply a Function over a List or Vector